conf/README.md

```diff
@@ -159,11 +165,12 @@ First install and configure nginx.  This is not covered in this document.
 Then set up the webwork2 Mojolicious app:
 
 * Copy `webwork2.mojolicious.dist.yml` to `webwork2.mojolicious.yml` if you want to modify settings in that file.
-* Copy `webwork2.nginx.dist.conf` to `webwork2.nginx.conf`.
+* Copy `webwork2.nginx.dist.conf` to `webwork2.nginx.conf`.  This will setup a reverse proxy for all
+webwork2 paths to proxy to the hypnotoad server. Generally this does not need to be changed.
 * Usually that files will not need to be modified.  You could even use the `.dist` file directly in most cases.
```

Comment:
This line should be deleted with your changes above.  It also has a typo.  "Usually that files" was meant to be "Usually that file", and is saying the same thing you added about "Generally this does not need to be changed."

Reply:
correct.